features/step_definitions/work_package_steps.rb

```diff
@@ -89,6 +89,18 @@
   end
 end
 
+When /^the work package table has finished loading$/ do
+  message <<-MESSAGE
+    This is a safeguard to ensure that the work package table is loaded before performing actions
+    on ui items that have not been fully loaded.
+
+    It currently assumes that at least on filter is set, without the necessity of the filter being
+    displayed.
+  MESSAGE
+
+  expect(page).to have_selector(".advanced-filters--filter", visible: false), message
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.